### PR TITLE
feat(embedremover): Remove embeds from all messages in a channel

### PR DESCRIPTION
**Pull request by jzyrobert**
If the feedback is accepted

### CODE REVIEW

**Comment by jzyrobert** [comment]
Can be done through discord perms

**Comment by jzyrobert** [comment]
Apparently didnt work with perms

File: unity_cogs/embedremover.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+
+class EmbedRemover(commands.Cog):
+    def __init__(self, client):
+        self.client = client
+
+    # Events
+    @commands.Cog.listener()
+    async def on_ready(self):
+        print('Renderdistance meme cog online')
```
Comment (bitshiftsdotio) [inline unity_cogs/embedremover.py:21]: Make sure the log message is correct. Eventually I will break this out into a utility decorator which does this automatically, but for now please change it.

**Review by bitshiftsdotio — CHANGES_REQUESTED** [review]

File: unity_cogs/embedremover.py (hunk 1 above)
Comment (bitshiftsdotio) [reply unity_cogs/embedremover.py:21]: Also, use `logging.info`

**Review by bitshiftsdotio — APPROVED** [review]

**Pull request merged**